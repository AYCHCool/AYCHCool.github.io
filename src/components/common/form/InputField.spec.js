import React from 'react';
import { shallow, mount } from 'enzyme';
import Input from './InputField';

const props = {
  type: '',
  name: '',
  value: '',
  onChange: jest.fn(),
};

describe('InputForm Component', () => {
  it('Should render without errors in debug mode', () => {
    const component = shallow(<Input {...props} />);
    expect(component).toMatchSnapshot();
  });

  it('Should render along with children componenet', () => {
    const component = mount(<Input {...props} />);
    expect(component).toMatchSnapshot();
    expect(component.find('input')).toHaveLength(1);
  });
});
