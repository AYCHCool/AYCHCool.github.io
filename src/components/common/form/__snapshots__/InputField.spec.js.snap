// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputForm Component Should render along with children componenet 1`] = `ReactWrapper {}`;

exports[`InputForm Component Should render without errors in debug mode 1`] = `ShallowWrapper {}`;
